installing/installing-preparing.adoc

```diff
@@ -273,5 +301,15 @@ xref:../installing/installing_platform_agnostic/installing-platform-agnostic.ado
 |
 |
 |
+|
+|
+|===
+
+.Special use cases
```

Comment:
The decision is yours, but I think I'd go about this differently.

Could you add "Edge computing" to one of the preceding tables? (Maybe not.  If I read carefully, I'd know, but I didn't.)

If you prefer to keep this table, then maybe set "Edge computing" in the left column and then "Single node" (or Node) in the column.